spec/requests/confirmation_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'GET registration/confirmation', type: :request do
+  let!(:user) { create(:user, confirmed_at: nil) }
+  subject(:confirmation) { get user_confirmation_path, params: params, as: :json }
```

Comment:
You can use subject without a name, if you find it easier you can leave it as you have done it.

```suggestion
  subject { get user_confirmation_path, params: params, as: :json }
```